azkaban-common/src/main/java/azkaban/executor/ExecutionOptions.java

```diff
@@ -43,6 +49,9 @@ public class ExecutionOptions {
   /* override dispatcher selection and use executor id specified */
   public static final String USE_EXECUTOR = "useExecutor";
   public static final int DEFAULT_FLOW_PRIORITY = 5;
+  public static final ImmutableList<String> DEFAULT_EXECUTION_RESTARTABLE_STATUS =
+      ImmutableList.of(Status.EXECUTION_STOPPED.name(), Status.FAILED.name());
+  public static final int DEFAULT_EXECUTION_RESTART_LIMIT = 2;
```

Comment:
What's the difference between this list and the one in Status.java class:
```
public static final ImmutableSet<Status> RESTARTABLE_STATUSES =
      ImmutableSet.of(Status.READY, Status.DISPATCHING, Status.PREPARING, Status.EXECUTION_STOPPED);
```
Is it possible to consolidate/unify the user experience when it comes to flow restart-ability?

Reply:
actually I think there's some ambiguity, let me re-organize them.